test/e2e/storage/persistent_volumes-local.go

```diff
@@ -66,18 +66,22 @@ const (
 	TmpfsLocalVolumeType localVolumeType = "tmpfs"
 	// tests based on local ssd at /mnt/disks/by-uuid/
 	GCELocalSSDVolumeType localVolumeType = "gce-localssd-scsi-fs"
+	// Creates a local file, formats it, and mounts it as a block device.
+	BlockLocalVolumeType localVolumeType = "block"
```

Comment:
I think you need to add "[Feature: LocalPersistentVolumes]" to all the block tests since block is still an alpha feature

Reply:
Ok. I'll add "[Feature: BlockVolume]" to block tests. If this isn't what you meant, please let me know.